js/reveal.js

```diff
@@ -3876,13 +3873,12 @@
             iframe.style.maxHeight = '100%';
             iframe.style.maxWidth = '100%';
 
-            // If not on current slide, this iframe has been lazy loaded.
-            // Need to add the data-lazy-loaded attribute so Reveal will handle it
-            // with all of its other lazy loaded content
-            if ( !isCurrentSlide ) {
-              iframe.setAttribute('data-lazy-loaded', '');
-            }
-
+            // Add the lazy-loaded tag to ensure that iframe will be
```

Comment:
Note that adding the `data-lazy-loaded` tag to all the background iframes (and not only the ones with `lazy-loaded`) ensure that they will be unloaded (`src` to `data-src`) once out of the viewDistance.
If not, any iframe added on the background will never be unloaded throughout the presentation.